man8/ssm.8

```diff
@@ -106,13 +106,13 @@ show this help message and exit
 show program\(aqs version number and exit
 .TP
 .B \-v\fP,\fB  \-\-verbose
-Show aditional information while executing.
+Show additional information while executing.
```

Comment:
You're changing a generated file. In this case there is no reason to change anything else since the changes will be pulled from ssmlib/main.py. See the doc/ for more information on how the documentation is generated in ssm.
Also I think that Jan made a mistake by including the ssm.8 in the git. I should fix it.

Reply:
fixed in latest commits